scripts/lint/main.ts

```diff
@@ -1,39 +1,38 @@
 import { REPO_ROOT } from "@/constants.ts";
 
 import { parseArgs } from "@std/cli";
-import * as fs from "@std/fs";
 import * as path from "@std/path";
 // @ts-types="npm:@types/less";
 import less from "less";
 
 import { checkForMissingFiles } from "@/lint/file-checker.ts";
 import { log } from "@/logger.ts";
 import { verifyMetadata } from "@/lint/metadata.ts";
-import { lint } from "@/lint/stylelint.ts";
-import { getUserstylesData } from "@/utils.ts";
+import { runStylelint } from "@/lint/stylelint.ts";
+import { getUserstylesData, getUserstylesFiles } from "@/utils.ts";
 import stylelintConfig from "../../.stylelintrc.js";
 
 const args = parseArgs(Deno.args, { boolean: ["fix"] });
-const subDir = args._[0]?.toString() ?? "";
-const stylesheets = fs.walk(path.join(REPO_ROOT, "styles", subDir), {
-  includeFiles: true,
-  includeDirs: false,
-  includeSymlinks: false,
-  match: [/\.user.css$/],
-});
+const userstyle = args._[0]?.toString().match(
+  /(?<base>styles\/)?(?<userstyle>[a-z0-9_\.]+)(?<trailing>\/)?(?<file>catppuccin\.user\.css)?/,
+)?.groups?.userstyle;
+const stylesheets = userstyle
+  ? [path.join(REPO_ROOT, "styles", userstyle, "catppuccin.user.css")]
+  : getUserstylesFiles();
```

Comment:
Some different logic so that a) we don't needlessly glob for files, we just look for subdirectories with catppuccin.user.css files in them, and that we b) are more lenient on what arguments you can pass to specify which indivudual userstyle you want linted - you can now do `styles/github/`, `styles/github/catppuccin.user.css`, or `github`, for example.